Streetcode/Streetcode/appsettings.IntegrationTests.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=_CHANGE_ME;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
Let's rename this "DefaultConnection" to "Main" and in future you will have to add here connection string to your database in docker container